src/main/java/net/snowflake/client/jdbc/SnowflakeDatabaseMetaData.java

```diff
@@ -137,6 +121,37 @@ private void raiseSQLExceptionIfConnectionIsClosed() throws SQLException {
     }
   }
 
+  /**
+   * Function to send in-band telemetry data about DatabaseMetadata get API calls and their
+   * associated SHOW commands
+   *
+   * @param functionName name of DatabaseMetadata API function call
```

Comment:
you missed queryId

Reply:
Now the function takes in ResultSet object instead of queryId. This way, the function can try to retrieve the queryId and insert the value in the telemetry data if it exists.